src/main/java/com/mixpanel/android/viewcrawler/ViewSnapshot.java

```diff
@@ -232,6 +234,21 @@ private void addProperties(JsonWriter j, View v)
         }
     }
 
+    // we only do layouts for views with RelativeLayout for now
+    private void addLayouts(JsonWriter j, View v)
+        throws IOException {
+        if (v.getParent() instanceof RelativeLayout) {
```

Comment:
I'd rather call getLayoutParams() and then check instanceof if we can- it's closer to what we mean.